lib/meadow/ingest/progress.ex

```diff
@@ -31,6 +31,25 @@ defmodule Meadow.Ingest.Progress do
     |> Repo.all()
   end
 
+  @doc """
+  Errors out all remaining pending entries for a row
+  """
+  def abort_remaining_pending_entries(%{ingest_sheet: ingest_sheet_id, ingest_sheet_row: row_id}) do
+    from(p in Progress,
+      as: :entry,
+      join: r in Row,
+      as: :row,
+      on: r.id == p.row_id,
+      where: r.row == ^row_id and p.status == "pending" and r.sheet_id == ^ingest_sheet_id
+    )
+    |> Repo.update_all(set: [status: "error"])
+  end
+
+  @doc """
+  UI upload or case w/out ingest sheet
+  """
+  def abort_remaining_pending_entries(_), do: :noop
```

Comment:
The compiler is complaining about duplicate `@doc`s – only one `@doc` per combination of function name + arity is allowed. Maybe convert the second one to a comment?